Unbox.swift

```diff
@@ -649,26 +710,26 @@ private extension UnboxableWithContext {
 }
 
 private extension Unboxer {
-    static func unboxerFromData(data: NSData, context: Any?) throws -> Unboxer {
+    static func unboxerFromData(data: NSData, context: Any?, skipInvalidData: Bool = false) throws -> Unboxer {
```

Comment:
Better to not use default values in the implementation, since it forces you to make a decision whether to include it or not (since the convenience aspect of it isn't really relevant for implementation details, only for public APIs).